cmd/sops/main.go

```diff
@@ -619,6 +618,19 @@ func main() {
 			log.Info("File written successfully")
 			return nil
 		}
+
+		if c.String("output") != "" {
+			file, err := os.Create(c.String("output"))
+			if err != nil {
+				return common.NewExitError(fmt.Sprintf("Could not open output file for writing: %s", err), codes.CouldNotWriteOutputFile)
+			}
+			defer file.Close()
+			_, err = file.Write(output)
+			if err != nil {
+				return toExitError(err)
+			}
+			log.Info("Output File written successfully")
+		}
 		_, err = os.Stdout.Write(output)
```

Comment:
Hmm, so this will write to stdout as well as the file. Maybe it'd be best to introduce a variable here that's an `io.Writer`. Something like this (no guarantee this will compile):

```go
var outputFile io.Writer
if c.String("output") != "" {
  // ...
  outputFile = file
} else {
  outputFile = os.Stdout
}
outputFile.Write(output)

Reply:
I think there should be a more elegant way of doing this

Reply:
Hmm, I'd think what I propose would be pretty idiomatic Go code. What makes you think otherwise? I'm certainly open to other options here!

Reply:
Also, now that I look at the code, I think this would be even better, because after this, we'll have three possible files to output to:

1. Standard out
2. The input file, with `--in-place`
3. The output file specified with `--output`

I'm not sure allowing combinations of these is a good idea. E.g. `sops --output bar.yaml --in-place --decrypt foo.yaml` would write the output to two different places.

Reply:
> I'm not sure allowing combinations of these is a good idea.

It definitely isn't and will surprise the user. If both `--in-place` and `--output` are set, we should exit with an error. If either are set, only logs should be displayed on the terminal.

Reply:
@jvehent what kind of error message should we print in case of the above mentioned situation?